services/src/main/java/org/keycloak/forms/login/freemarker/FreeMarkerLoginFormsProvider.java

```diff
@@ -150,6 +152,7 @@ public FreeMarkerLoginFormsProvider(KeycloakSession session) {
         this.realm = session.getContext().getRealm();
         this.client = session.getContext().getClient();
         this.uriInfo = session.getContext().getUri();
+        this.lang = "en";
```

Comment:
Not sure if this is the same value, but perhaps we can use the [`Locale.ENGLISH`](https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html#ENGLISH) constant?